client/me/purchases/manage-purchase/index.jsx

```diff
@@ -741,10 +803,15 @@ class ManagePurchase extends Component {
 						/>
 					) }
 					{ isProductOwner && (
-						<>
+						<div className="manage-purchase__renew-upgrade-buttons">
 							{ preventRenewal && this.renderSelectNewButton() }
-							{ ! preventRenewal && this.renderRenewButton() }
-						</>
+							{ ! preventRenewal && (
```

Comment:
Does it make sense to hide the upgrade button conditionally based on the capability to renew?

Reply:
Yeah 🤔 On further thought, even if someone can't renew they might want to upgrade. I'll remove the conditional here.

Reply:
Another possible case for us to consider, though: is it appropriate for the "Select a new plan" button to appear alongside the "Upgrade" button?